synapse/handlers/federation.py

```diff
@@ -1090,12 +1154,14 @@ async def _maybe_backfill_inner(
         #   types have.
 
         forward_events = await self.store.get_successor_events(list(extremities))
+        logger.info("_maybe_backfill_inner: forward_events %s", forward_events)
```

Comment:
Currently, the insertion events get filtered out here because no events reference the insertion event as a `prev_event`.

When we receive a marker event, we can just add it to `insertion_event_extremity` and pull from that table here. Then just have the normal backfill mechanism pull it in.